test/k8s-integration/main.go

```diff
@@ -333,6 +333,8 @@ func generateTestSkip(normalizedVersion string) string {
 		// bug-fix introduced in 1.17
 		// (https://github.com/kubernetes/kubernetes/pull/81163)
 		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
```

Comment:
Side issue: This bug fix is actually in 1.16, so we can enable this test case in 1.16.

Reply:
Should we track this in a different issue?

Reply:
yes can be tracked separately